framework/test/mocha/functional/http/get/accounts/multisignatures.js

```diff
@@ -72,8 +72,7 @@ describe('GET /api/accounts', () => {
 		);
 
 		describe('address', () => {
-			// eslint-disable-next-line
-			it.skip('[feature/improve_transactions_processing_efficiency] using known address should respond with its multisignature_group', async () => {
+			it('using known address should respond with its multisignature_group', async () => {
```

Comment:
maybe rephrase it to
`it should respond with it's multisignature group when using known address`?
same for below

Reply:
Done